source/support/data-protection/index.html.md.erb

```diff
@@ -0,0 +1,162 @@
+
+# Types of requests
+
+There are different types of requests we may receive. The section below describes a user's rights and actions we may have to take in response to them.
+
+## Requests to access data
+
+Also been known as Subject Access Requests. A user with an account has a right to see information about them held on the system. This can help a user better understand how their data is being use, and provide a way check we are
+meeting promises we made in the [govuk-accounts-privacy-policy][2].
+
+User support may first get in contact to ask if we hold data for a particular person. They may be identified by an email address, or another piece of data we hold on the system.
+
+When user support contact us we should:
+- Confirm with user support we are looking into it
+- Open a new trello ticket using the [access request template][3]
+- Assign a developer to confirm if we hold data on that user
+- Respond to user support confirming if we have data within 3 days.
+
+If we do hold data, user support may return to ask if we can export it for a user. They will agree a format, and scope.
+
+If the request returns, reassign the ticket to a developer and ask them to export the data.
+
+@TODO: How should a developer export it, how is best to securely send it to user support, how do we securely send it to the individual?
+
+When asked by user support to provide a users data, we should respond to them with the data within 7 days.
+
+## Requests to erase data
+
+Users may request that we delete a portion of their account, or remove their account entirely. This is not an absolute right, there may be some circumstances where we may refuse to completely remove data. Privacy office will be able to advise in those circumstances.
+
+Where a user wishes to delete their entire account, user support will initially respond with our [planned response on how to delete your account][3].
+
+If a user does not wish to log in to delete their account, user support will contact the accounts team.
+
+When user support contact us we should:
+- Confirm with user support we are looking into it
+- Open a new trello ticket using the [manual erasure template][4]
+- Assign a developer to delete the account.
+- Confirm to user support that we have deleted the data within 7 days.
+
+We only need to be concerned with data held in accounts. Data such as email subscriptions will be handled by other teams within GOV.UK.
+
+Users also have a right to have just a portion of their data deleted. If a user requests this and does not wish their account to be deleted we should discuss an appropriate response as a team and update this guide then.
+
+Presently all data on the account is either integral to its functioning (e.g. MFA data, email address), or is the central reason to use an account (e.g. Brexit checker results).
+
+Security events will not be removed during this process, however they will no longer be identifiable to a user.
```

Comment:
Are security events user data as well?

Reply:
Whilst a user account exists they are associated.
This is how we present the user with their list of events on their dashboard.

However once we've deleted the account that association is lost and they just become free floating sets of anonymous logs. So we know _someone_ logged in or changed their details for instance, but no longer have a way of finding out who that was.